src/main/java/com/cloudbees/hudson/plugins/folder/relocate/RelocationAction.java

```diff
@@ -168,6 +171,10 @@ public Object getStaplerFallback() {
 
         @Override
         public Collection<? extends Action> createFor(@Nonnull Item target) {
+            if (target.getParent() instanceof ComputedFolder) {
+                // cannot move from a computed folder
```

Comment:
I want to ensure that once we allow for name mangling we prevent the risk of moving items